modules/calib3d/test/test_undistort.cpp

```diff
@@ -1807,4 +1807,79 @@ TEST(Calib3d_initUndistortRectifyMap, regression_14467)
     EXPECT_LE(cvtest::norm(dst, mesh_uv, NORM_INF), 1e-3);
 }
 
+TEST(Calib3d_initInverseRectificationMap, regression_14467)
+{
+    Size size_w_h(512 + 3, 512);
```

Comment:
It makes sense to use a really different width/height. Use some regular resolution like 640x480 or 1280x720 (camera matrix should be updated too)

Reply:
Updated intrinsics and extrinsics specification to use experimentally collected values. 
```
Size size_w_h(1280, 800);
```
```
// Camera Matrix
double k[9]={
    1.5393951443032472e+03, 0., 6.7491727003047140e+02,
    0., 1.5400748240626747e+03, 5.1226968329123963e+02,
    0., 0., 1.
};
```

```
double d[5]={ // Non-zero distortion
    -3.4134571357400023e-03, 2.9733267766101856e-03, // K1, K2
    3.6653586399031184e-03, -3.1960714017365702e-03, // P1, P2
    0. // K3
};
```

Note, K1 and K2 have been increased from `-3.4134571357400023e-01, 2.9733267766101856e-01` to meet error threshold of ` 2e-1`